frontend/packages/knative-plugin/src/utils/create-channel-utils.ts

```diff
@@ -36,10 +36,10 @@ export const getChannelKind = (ref: string): string => {
 
 export const useChannelList = (namespace: string): ChannelListProps => {
   const [accessData, setAccessData] = useSafetyFirst({ loaded: false, channelList: [] });
-  const channelResourcesList = getDynamicChannelModelRefs();
-  const accessList = [];
+  const channelResourcesList = useChannelResourcesList();
```

Comment:
better to destructure loaded and channels here

Reply:
Updated.